src/components/Library/LibrarySearch/LibraryCard/LibraryCard.jsx

```diff
@@ -132,9 +132,7 @@ const LibraryCard = (props) => {
           style={contentContainerStyle}
         >
           <ContentTextarea
-            className="library-card-textarea"
-            isSelected={isSelected}
-            lib={true}
+            className={`library-card-textarea ${isSelected ? "selected" : ""}`}
```

Comment:
Could you pull out the code from ContentTextarea into LibraryCard? (And delete ContentTextarea).